src/System.Private.CoreLib/shared/System/Runtime/Intrinsics/X86/Avx2.PlatformNotSupported.cs

```diff
@@ -455,7 +455,7 @@ public static class Avx2
         /// __m128i _mm256_extracti128_si256 (__m256i a, const int imm8)
         ///   VEXTRACTI128 m128, ymm, imm8
         /// </summary>
-        public static unsafe void ExtractVector128(sbyte* address, Vector256<sbyte> value, byte index) { throw new PlatformNotSupportedException(); }
+        new public static unsafe void ExtractVector128(sbyte* address, Vector256<sbyte> value, byte index) { throw new PlatformNotSupportedException(); }
```

Comment:
We expose both AVX and AVX2 versions of these functions...

The AVX version uses `VEXTRACTF128` (@fiigii, does this incur any kind of stall?) while the AVX2 version uses `VEXTRACTI128`.

We should determine if we want to:
* Expose as is
* Expose only on AVX and document that it will use `VEXTRACTI128` on AVX2 hardware
* Don't expose these overloads on AVX, and require users reinterpret-cast to `float` or `double`

Reply:
> does this incur any kind of stall?

Yes, but just a little bit. Usually, that is just one cycle.